### DISCUSSION TITLE
[Unstable] Massive FPS drop, save/video included

### DISCUSSION THREAD

**Discussion by Fartnite3** [FakeFishGames/Barotrauma#14738] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Massive random fps drops from stable 144 to around 20 when hosting an unstable server, happened in both a sandbox mission in herja, and a campaign in azimuth. no mods 

### Reproduction steps

happens completely randomly, can occur in mission or while docked, according to task manager barotrauma is using around 2% of the cpu while the drops occur, hosted in multiplayer with and without other players in lobby. bug has not appeared on stable release.
[.unst test.save.zip](https://github.com/user-attachments/files/17153861/default.unst.test.save.zip)

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by lIllIIlI** [comment]
Try using showperf and/or record a video

**Reply by Fartnite3** [reply]
https://www.youtube.com/watch?v=1SY2um6X72U 

couldn't compress the clip to 10mb without tanking the quality

**Reply by Fartnite3** [reply]
issue starts at 0:30

**Comment by itchyOwl** [comment]
There's definitely some performance issue there, but the profiler doesn't seem to point to anything specific. I'll create a ticket about this.

**Comment by Fartnite3** [comment]
As of the latest unstable update the issue cannot be replicated, thanks for the quick fix :)
